translator/config/schema.json

```diff
@@ -100,6 +100,16 @@
           "minLength": 1,
           "maxLength": 255
         },
+        "global_dimensions": {
+          "type": "object",
+          "description": "Any dimensions to be applied to all published metrics",
+          "maxProperties": 30,
```

Comment:
I wonder if we should limit the global dimensions. Since telegraf as default for each plugin will have the [default tag (e.g cpu will have total_cpu, cpu_core)](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/cpu). Therefore, for avoid the global_dimensions override those one, would it be better to limit these here?

Reply:
It really depends on which Telegraf metric you are talking about; some have additional tags, some don't. I suspect that 30 is _way_ more than anyone is likely to need though so limiting to something smaller (e.g. 20?) might well make sense.

Reply:
I'd rather just keep it at 30 to be consistent with API guardrails than trying to reason a number in between